package.json

```diff
@@ -61,6 +61,7 @@
     "localforage": "^1.7.2",
     "magento2-rest-client": "github:DivanteLtd/magento2-rest-client",
     "pm2": "^2.10.4",
+    "redis-tag-cache": "^1.2.1",
```

Comment:
why do we need this dependency here? I cannot find any reference in PR to that

Reply:
no idea, shouldn't be there